### ISSUE TITLE
CQL Tracing: Inform the amount of elements trimmed off from a list

### ISSUE THREAD

**Issue by raphaelsc** [scylladb/scylla#7319]
Tracing denotes that a list of values was trimmed with three dots (only up to 64 bytes will be logged), for example:
`{'consistency_level': 'QUORUM', 'page_size': '5000', 'param[0]': '1', 'param[1]': '4', 'param[2]': '1', 'param[3]': '15655', 'param[4]': '202011, 202012, 202013, 202014, 202015, 202016, 202017, 202018, ...', `

but it's useful to know the amount of elements trimmed off from a list, like when looking for bad queries, containing many elements, that cause performance issues. Something like the follow:

`{'consistency_level': 'QUORUM', 'page_size': '5000', 'param[0]': '1', 'param[1]': '4', 'param[2]': '1', 'param[3]': '15655', 'param[4]': '202011, 202012, 202013, 202014, 202015, 202016, 202017, 202018, ... (10 elements were left out)', `

**Comment by raphaelsc** [comment]
@vladzcloudius FYI

**Comment by vladzcloudius** [comment]
@raphaelsc I agree that we need to be able to know the total amount of IN elements from the trace.

I'm not sure that this info deserves a special treatment in the `sessions` table which is supposed to be as generic a and laconic as possible.

We may want to add the corresponding trace point that would log this info: trace points are saved in the `events` table.

BTW, could you share the corresponding `events` table content: SELECT * FROM system_traces.events where session_id=<session ID in question>` ?
